src/containers/Organization/Onboarding/Steps/OrgDetails.tsx

```diff
@@ -0,0 +1,148 @@
+import { Input } from 'components/UI/Form/Input/Input';
+import * as Yup from 'yup';
+import { FormLayout } from '../FormLayout/FormLayout';
+import { useState } from 'react';
+
+import styles from '../FormLayout/FormLayout.module.css';
+import { ONBOARD_URL_UPDATE } from 'config';
+import axios from 'axios';
```

Comment:
Can you please reorder the imports to include external packages first followed by internal ones.